ignite/metrics/accuracy.py

```diff
@@ -14,9 +14,11 @@ def __init__(
         self,
         output_transform: Callable = lambda x: x,
         is_multilabel: bool = False,
+        is_variable_multiclass: bool = False,
```

Comment:
can't we rename this to something like `variable_classes`? We already have `is_multilabel` here and together they should be the same,  but maybe we can reuse `variable_classes` in some further way.

So `is_variable_multiclass` would be the same as `is_multilabel and variable_classes`. This won't change the functionality though but we don't have separate variables indicating the same thing (partially)

Reply:
`variable_classes` sounds good. However, we need emphasize that this is possible only for multiclass case and not multi-label.

Reply:
sounds good. I pushed the changes